src/index.tsx

```diff
@@ -1,14 +1,18 @@
+import React from "react";
+import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 import { initializeIcons } from "@fluentui/react";
-import { createRoot } from "react-dom/client";
 
 initializeIcons();
 
-const container = document.getElementById("root");
-const root = createRoot(container!);
-root.render(<App />);
+ReactDOM.render(
+    <React.StrictMode>
+        <App />
+    </React.StrictMode>,
+    document.getElementById("root"),
+);
```

Comment:
Das sollte wohl zurückgedreht werden, es produziert zumindest den folgenden Fehler in der Konsole:
`Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot`